src/credentials/FabricTable.cpp

```diff
@@ -538,6 +538,32 @@ const FabricInfo * FabricTable::FindFabricWithCompressedId(CompressedFabricId co
     return nullptr;
 }
 
+FabricInfo * FabricTable::FindFabricWithRootPubKeyFabricId(const Crypto::P256PublicKey & rootPubKey, const FabricId & fabricId)
```

Comment:
Also, we cannot have any public methods on FabricTable that give a non-const `FabricInfo *` back, since that breaks the shadow data maintenance from ongoing UpdateNOC/AddNOC

Reply:
Ah, I had implemented this against an old version of the sdk.

Will remove.  Thanks.